### PR TITLE
Tag 49

### PR DESCRIPTION
**Pull request by flapek**
Added role to user and policy to authorization.

### CODE REVIEW

**Review by flapek — COMMENTED** [review]
Self check done

**Review by kacperantoniak — APPROVED** [review]

File: TagsterWebAPI/src/Tagster.Infrastructure/EF/TagsterDbContext.cs [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
     public DbSet<Tag> Tags { get; set; }
     public DbSet<Profile> Profiles { get; set; }
     public DbSet<User> Users { get; set; }
     public DbSet<RefreshToken> RefreshTokens { get; set; }
 
-    public TagsterDbContext(DbContextOptions options) : base(options)
+    public TagsterDbContext(DbContextOptions options, IPasswordService passwordService) : base(options)
     {
+        _passwordService = passwordService;
+        DefaultUsers = CreateDefaultUsers();
```
Comment (kchowanski) [inline TagsterWebAPI/src/Tagster.Infrastructure/EF/TagsterDbContext.cs:23]: do we need this call and assignment in ctor? As I can see we use it to generate migration only so calling it from OnModelCreating should be enough. By doing this we can also remove DefaultUsers field declaration.

File: TagsterWebAPI/src/Tagster.Application/Commands/ChangeUserRole/ChangeUserRole.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Tagster.Application.Commands.ChangeUserRole;
+
+public class ChangeUserRole
```
Comment (kchowanski) [inline TagsterWebAPI/src/Tagster.Application/Commands/ChangeUserRole/ChangeUserRole.cs:5]: where I can find handler for this command?

Comment (flapek) [reply TagsterWebAPI/src/Tagster.Application/Commands/ChangeUserRole/ChangeUserRole.cs]: I forgot add it. But now it is in repository

File: TagsterWebAPI/src/Tagster.Infrastructure/EF/TagsterDbContext.cs (hunk 1 above)
Comment (flapek) [reply TagsterWebAPI/src/Tagster.Infrastructure/EF/TagsterDbContext.cs]: I did small refactor

**Review by kchowanski — APPROVED** [review]

**Pull request merged**